docs/quality-assurance/security-testing-strategy.md

```diff
@@ -0,0 +1,66 @@
+# UKHO Security Testing Strategy
+*Note: This document is the property of the United Kingdom Hydrographic Office and may contain third party material.  It must not be copied or transmitted to any third parties without the written permission of the Owner(s).  :copyright: Crown Copyright 2019*
+
+## Introduction
+The purpose of this document is to outline, at a high level, how Security Testing will be carried out across the UKHO. This will include the tools and technologies used across the various stages of the software development lifecycle.
+
+The Test Community focus will be on Web Application Penetration Testing.
```

Comment:
You say "Web Application Penetration Testing" but then don't mention it again. How does it relate to the bits below.